src/makeInfinityAdder.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function(x) {
+    if (x === undefined) {
+      return 0;
+    }
+    sum += x;
```

Comment:
this function should return current sum(not 0) when x is not passed and set 0 to sum